maps/DeadSpace/M.03 Aegis VII.dmm

```diff
@@ -48,7 +48,6 @@
 "aV" = (/obj/machinery/alarm{dir = 8; pixel_x = 25; pixel_y = 0},/turf/simulated/floor/tiled,/area/space)
 "aW" = (/obj/item/modular_computer/console/preset/security{icon_state = "console"; dir = 4},/obj/effect/floor_decal/industrial/outline/yellow,/turf/simulated/floor/tiled/monotile,/area/space)
 "aX" = (/obj/machinery/door/airlock/civilian{dir = 8},/obj/machinery/door/firedoor/border_only{icon_state = "open"; dir = 4},/obj/structure/cable/green{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{dir = 4},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/turf/simulated/floor/tiled/monotile,/area/space)
-"aY" = (/obj/structure/cable/green{d2 = 4; icon_state = "0-4"},/obj/structure/table/reinforced,/obj/item/weapon/folder/red,/obj/item/weapon/stamp/denied,/obj/item/weapon/stamp/brig,/turf/simulated/floor/tiled,/area/space)
```

Comment:
why are mapping changes mixed in?

Reply:
Mapping changes are to fix an error on the Aegis colony map. It will not cause a conflict since I'm not working on it currently on the mapping branch.